<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>serial</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>571</width>
      <height>171</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="pushButton_clearInformation">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>10</y>
       <width>112</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Information</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>46</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DTR</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>46</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RTS</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>364</x>
       <y>14</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Time</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_sendHexStr">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Send Hex String</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_showInHex">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>14</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Show in Hex</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_sendWithEnter">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Send with Enter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sendData">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>51</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Send Data</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>13</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>COM Port:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_port">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>11</y>
       <width>57</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_openPort">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>70</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Port</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>141</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_stopBits">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>StopBits:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_baudRate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Baudrate:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_parity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>42</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Parity:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_stopBits">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>57</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_byteSize">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>71</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ByteSize:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_byteSize">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>57</width>
        <height>20</height>
       </rect>
      </property>
      <property name="currentText">
       <string>4</string>
      </property>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_flowControl">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>57</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SW</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>HW</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_baudRate">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>57</width>
        <height>20</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>230400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>460800</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_parity">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>57</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Odd</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Even</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mark</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Space</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_flowControl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>67</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>FlowControl:</string>
      </property>
     </widget>
    </widget>
    <widget class="QTextEdit" name="textEdit_send">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>414</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>133</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Log</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_autoSend">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>46</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Timing transmission</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_timeTrans">
     <property name="geometry">
      <rect>
       <x>386</x>
       <y>46</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_ms">
     <property name="geometry">
      <rect>
       <x>452</x>
       <y>47</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>228</x>
       <y>131</y>
       <width>336</width>
       <height>22</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>log file path</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_receive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>571</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
